File: rad_models/raddaily.js
module.exports = function (sequelize, _DataTypes) {
  return sequelize.define(
    "raddaily",
    {
      id: {
        type: "INT(11)",
        allowNull: false,
        defaultValue: null,
        primaryKey: true,
        autoIncrement: true,
        comment: null,
        foreignKey: [Object],
      },
      username: {
        type: "VARCHAR(64)",
        allowNull: false,
        defaultValue: null,
        primaryKey: false,
        autoIncrement: false,
        comment: null,
        foreignKey: [Object],
      },
      usageorig: {
        type: "BIGINT(20)",
        allowNull: false,
        defaultValue: null,
        primaryKey: false,
        autoIncrement: false,
        comment: null,
      },
      usagediscount: {
        type: "BIGINT(20)",
        allowNull: false,
        defaultValue: null,
        primaryKey: false,
        autoIncrement: false,
        comment: null,
      },
      createddate: {
        type: "DATE",
        allowNull: false,
        defaultValue: null,
        primaryKey: false,
        autoIncrement: false,
        comment: null,
        foreignKey: [Object],
      },
    },
    {
      tableName: "raddaily",
    }
  );
};

File: internet/model/raddaily.py
from sqlalchemy.orm import Mapped, mapped_column
from datetime import date

from . import Base


# pyre-ignore[11]
class RadiusDaily(Base):
    __tablename__ = "raddaily"

    id: Mapped[int] = mapped_column(primary_key=True)
    username: Mapped[str]
    usage_original: Mapped[int] = mapped_column("usageorig")
    usage_discount: Mapped[int] = mapped_column("usagediscount")
    create_date: Mapped[date] = mapped_column("createddate")
